search/services/index.py

```diff
@@ -121,7 +121,7 @@ def __init__(self, host: str, index: str, port: int=9200, **extra) -> None:
 
     @staticmethod
     def _get_operator(obj):
-        if type(obj) is tuple:
+        if isinstance(obj, tuple):
```

Comment:
So this is where pylint and I part ways. IIRC isinstance will return True not just for type(obj) is tuple, but also when obj is an instance of a subclass of tuple. Here and elsewhere, my assumption would be that we really want to know whether obj is a tuple, and not just an instance of a subclass of a tuple.